### ISSUE TITLE
Warning: Illegal string offset 'show' in /home3/wacpg60abqhg/public_html/wp-content/themes/flatsome/inc/shortcodes/ux_products.php on line 82 

### ISSUE THREAD

**Issue by 1sterlingfox** [woocommerce/woocommerce#17295]
## Prerequisites

- [ ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [ ] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [ ] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1.
2.
3.

## Expected/actual behavior

When I follow those steps, I see...

I was expecting to see...

## Isolating the problem

- [ ] This bug happens with only WooCommerce plugin active
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ ] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin here.
```
</details>

**Comment by willgorham** [comment]
@1sterlingfox This error is coming from your theme. Have you tried contacting your theme author?

**Comment by spencerfinnell** [comment]
Duplicate: https://github.com/woocommerce/woocommerce/issues/17283

"Hi there! This form is for reporting bugs and issues specific to the WooCommerce plugin. This is not a support portal. If you need technical support from a human being, please submit a ticket via the helpdesk instead: https://woocommerce.com/contact-us/"

As mentioned above the error is coming from your theme. Contact your theme author to ensure compatibility with the latest version of WooCommerce.

You also failed to complete the issue submission form which will make getting help even less likely.

**Comment by 1sterlingfox** [comment]
I appreciate your reply and i also understand that it says its in the theme. i contacted the author of the theme and they are trying to get admin access to my website wich i feel uncomfortable with. also if i deactivate the woocommerce plug in the code goes away. so i am not really sure what to do 
Sterling Fox 

    On Friday, October 20, 2017, 9:07:15 AM PDT, Mike Jolley <notifications@github.com> wrote:  
 
 
Closed #17295.

**Comment by mikejolley** [comment]
If you don't want to give access to the developer, I don't see how anyone else can help. It's definitely coming from the theme - it says so in the error message. Of course turning off WooCommerce will hide the error, because the theme will no longer be loading that code..
